src/Data/AudienceFolder.elm

```diff
@@ -23,6 +30,44 @@ type alias AudienceFolder =
     }
 
 
+-- = BASIC FIELDS DECODERS =
+idField = D.field "id" D.int
+nameField = D.field "name" D.string
+parentField = D.field "parent" (D.nullable D.int)
```

Comment:
Not a fan of `elm-format`? :slightly_smiling_face:

Reply:
I didn't know it existed. But it seems that vim installation is pretty easy, so I wouldn't have any problem using it.